src/PaymentGateways/Gateways/Stripe/ValueObjects/DonationSummary.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\ValueObjects;
+
+use Give\PaymentGateways\DataTransferObjects\GatewayPaymentData;
+
+class DonationSummary
```

Comment:
@kjohnson  In most payment gateway, we send donation summary for gateway payment. I can see that this will be useful if we move this class to `framework/Paymentgateways`.

What do you think?

Reply:
My expectation is that the `DonationSummary` gets extracted into a shared Gateway API component. That said, I have chosen to introduce it as a part of the Stripe gateway as a first iterative step.